redis_store.go

```diff
@@ -178,7 +178,7 @@ func (this *RedisStore) Poll(c chan []byte, queue string) error {
 				continue
 			}
 
-			message, err := redis.Bytes(consumer.Do("LPOP", queue))
+			message, err := redis.Bytes(consumer.Do("BLPOP", queue, 10))
```

Comment:
Sorry, I forgot to tell you about this change. A few days ago I noticed that Incus was flooding Redis with a lot of pop requests so I made this change to improve efficiency. And yes, you are right, this way Incus will not continuously try to pop the list if it is empty.